targets/TARGET_NUVOTON/TARGET_NUC472/crypto/crypto-misc.c

```diff
@@ -216,7 +216,11 @@ static bool crypto_submodule_acquire(uint16_t *submodule_avail)
 static void crypto_submodule_release(uint16_t *submodule_avail)
 {
     uint16_t expectedCurrentValue = 0;
-    while (! core_util_atomic_cas_u16(submodule_avail, &expectedCurrentValue, 1));
+    while (! core_util_atomic_cas_u16(submodule_avail, &expectedCurrentValue, 1)) {
```

Comment:
Aren't we supposed to write `0` here?

Reply:
@kjbracey-arm Yes, this loop is unnecessary. Anyway, semaphore is used to substitute for busy-wait loop.